app/models/branch.rb

```diff
@@ -69,7 +69,8 @@ def conditionals_attributes=(hash)
   end
 
   def destinations
-    destinations_list(flow_objects: ordered_flow)
+    all_flow_objects = ordered_flow + detached
+    destinations_list(flow_objects: all_flow_objects)
```

Comment:
Do we have tests for the destinations method and detached objects?

Reply:
will add them :)